docs/config-scale.html

```diff
@@ -988,12 +988,12 @@ <h3 class="item-subtitle">
     </td>
     <td class="size-value">
       <code>
-        0.9rem
+        .9rem
```

Comment:
@mirisuzanne I'm not 100% sure what's causing this change in leading zeros... I assume an upgrade in node-sass version?

Reply:
yeah, not sure exactly - but likely just a (non-breaking) change between Sass implementations.